dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java

```diff
@@ -110,6 +121,26 @@ public int getFirstRefreshInterval() {
     return finalConfig.getInt(FIRST_REFRESH_INTERVAL, DEFAULT_FIRST_REFRESH_INTERVAL);
   }
 
+  public Boolean isProxyEnable() {
+    return finalConfig.getBoolean(PROXY_ENABLE, false);
+  }
+
+  public String getProxyHost() {
+    return finalConfig.getString(PROXY_HOST, "127.0.0.1");
+  }
+
+  public int getProxyPort() {
+    return finalConfig.getInt(PROXY_PORT, 8080);
+  }
+
+  public String getProxyUsername() {
+    return finalConfig.getString(PROXY_USERNAME, "user");
```

Comment:
It's better to use an empty user and password when not there

Reply:
I've set the default value of PROXY_USER and PROXY_PASSWD to "" now, please review